src/main/java/hudson/tasks/junit/CaseResult.java

```diff
@@ -51,11 +51,14 @@
 import java.util.Collections;
 import java.util.Comparator;
 import java.util.List;
-import java.util.logging.Logger;
+import java.util.logging.Level;import java.util.logging.Logger;
 
 import static java.util.Collections.emptyList;
 import static java.util.Collections.singletonList;
 
+import javax.xml.stream.*;
```

Comment:
@basil is it something to be concerned about introducing `javax` here? With the jakarta move coming up soon?

Reply:
I think its okay, these APIs appear to be in Java itself and not Java EE https://docs.oracle.com/en/java/javase/17/docs/api/java.xml/javax/xml/stream/XMLEventReader.html